numpy/core/setup.py

```diff
@@ -853,6 +855,7 @@ def get_mathlib_info(*args):
             join('src', 'multiarray', 'dragon4.c'),
             join('src', 'multiarray', 'dtype_transfer.c'),
             join('src', 'multiarray', 'einsum.c.src'),
+            join('src', 'multiarray', 'einsum_sumprod.c.src'),
```

Comment:
```suggestion
            join('src', 'multiarray', 'einsum.dispatch.c.src'),
```

Reply:
using `einsum.dispatch.c.src` instead of `einsum_prod.dispatch.c.src`

Reply:
yes, the rename is important because the review process is count on it.(all of the main code that changed is lied in `einsum.dispatch.c.src` )

Reply:
There is no need to rename in this PR. Git can track renames just fine, what it struggles with is files that split in two.